### PR TITLE
add happy token alias

### PR DESCRIPTION
**Pull request by wj-Mcat**
Fix: https://github.com/wechaty/python-wechaty/issues/194

### CODE REVIEW

File: src/wechaty_puppet_service/config.py [hunk 1]
```diff
@@ -19,8 +19,22 @@
...
+logger = get_logger('WechatyPuppetServiceConfig')
+
+
+def get_token():
+    """
+    get the token from environment variable
+    """
+    return os.environ.get('WECHATY_PUPPET_SERVICE_TOKEN', None) or \
+        os.environ.get('TOKEN', None) or \
+        os.environ.get('token', None) or None
```
Comment (wj-Mcat) [inline src/wechaty_puppet_service/config.py:34]: Get token from more friendly environment variable names.

File: tests/test_token_envrionment.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    token = 'your-self-token'
+    os.environ['token'] = token
+    assert get_token() == token
+
+
+def test_none_token():
+    del os.environ['WECHATY_PUPPET_SERVICE_TOKEN']
+    del os.environ['TOKEN']
+    del os.environ['token']
+    assert get_token() is None
```
Comment (wj-Mcat) [inline tests/test_token_envrionment.py:31]: Add Simple token test for `get_token` method.

**Review by wj-Mcat — COMMENTED** [review]
Two enhancements

**Comment by wj-Mcat** [comment]
Need to improve docs: 
* https://github.com/wechaty/python-wechaty-getting-started
* https://github.com/wechaty/python-wechaty-puppet-service

**Comment by huan** [comment]
Please keep the docs for using `WECHATY_PUPPET_SERVICE_TOKEN` as the main environment variable to keeping it consistent with TS and other Polyglot Wechaty (and also the long name makes it clear for explaining itself), and the new happy `TOKEN` and `token` should be documented as an **alias**.

Thank you very much.

**Comment by wj-Mcat** [comment]
@huan  Of course, I will keep the happy names as alias.

**Pull request merged**